activejob/README.md

```diff
@@ -3,7 +3,7 @@
 Active Job is a framework for declaring jobs and making them run on a variety
 of queuing backends. These jobs can be everything from regularly scheduled
 clean-ups, to billing charges, to mailings. Anything that can be chopped up into
```

Comment:
"Anything that can be chopped up into small units of work and run in parallel" is a sentence fragment (i.e. incomplete sentence).  The ", really" makes the tone informal / conversational, and serves to acknowledge the incomplete sentence as a stylistic choice.

If we remove the ", really", we should merge the fragment into the previous sentence:

```suggestion
clean-ups, to billing charges, to mailings — anything that can be chopped up into
```

Reply:
@jonathanhefner suggested changes updated